Quaver/Quaver.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
Not sure why the output type is being changed. By changing it from WinExe to Exe a console window is showing up on game startup.